site/app/libraries/database/AbstractDatabase.php

```diff
@@ -172,11 +172,6 @@ public function query($query, $parameters = []) {
             elseif ($identity === QueryIdentifier::SELECT) {
                 $columns = $this->getColumnData($statement);
                 $this->results = $statement->fetchAll(\PDO::FETCH_ASSOC);
-                // Under normal circumstances, we don't really need to worry about $this->results being false.
-                // @codeCoverageIgnoreStart
-                if ($this->results === false) {
-                    return false;
-                }
                 // @codeCoverageIgnoreEnd
```

Comment:
should this end comment also be removed

Reply:
Yeah I think it should. Missed that before. Thanks for catching it.